sysutils/scp-backup/src/opnsense/mvc/app/controllers/OPNsense/ScpBackup/forms/general.xml

```diff
@@ -0,0 +1,39 @@
+<form>
+  <field>
+    <id>general.enabled</id>
+    <label>Enabled</label>
+    <type>checkbox</type>
+    <help>This will activate the SCP backup plugin</help>
+  </field>
+  <field>
+    <id>general.hostname</id>
+    <label>Hostname</label>
+    <type>text</type>
+    <help>Set the remote hostname</help>
+  </field>
+  <field>
+    <id>general.port</id>
+    <label>Port</label>
+    <type>text</type>
+    <help>Set the remote port (if blank, defaults to port 22)</help>
+  </field>
+  <field>
+    <id>general.username</id>
+    <label>Remote Username</label>
+    <type>text</type>
+    <help>Set the remote username (if blank, defaults to 'root')</help>
+  </field>
+  <field>
+    <id>general.remoteLocation</id>
+    <label>Remote Location</label>
+    <type>text</type>
+    <help>Set the remote location to backup the config file to (if blank, defaults to './')</help>
+  </field>
+  <field>
+    <id>general.publicKey</id>
+    <label>Local Root's Public Key</label>
```

Comment:
Should not be the one of root or anyone else. Should be an autogenerated key for this service. Should also support passwords as well.

Maybe the fingerprint should be configurable as well for the remote server.

Reply:
This was agreed with Fitch.

Happy to accept PR's for passwords, fingerprints etc...